docs/src/pages/_app.page.tsx

```diff
@@ -19,6 +20,9 @@ function MyApp({ Component, pageProps }) {
   const [colorMode, setColorMode] = React.useState<ColorMode>('system');
   const [themeOverride, setThemeOverride] = React.useState('');
 
+  configure();
```

Comment:
This has a check to see if it is configured yet, so running it for each page render is ok?